amp-pwa-reader/src/js/Card.js

```diff
@@ -214,31 +207,47 @@ class Card {
   }
 
   activate() {
-
     // set main view to inert so you can't tab into it
     shadowReader.disableCardTabbing();
 
+    this.article.getClonedCardElem();
+
     // add loading spinner (and promote to layer)
     this.elem.classList.add('sr-loading', 'sr-promote-layer');
 
-    this.article.load()
-      .then(() => this.article.render())
-      .then(() => {
-        // remove loading spinner
+    // in streaming case, things happen in a different order
+    if (this.streaming) {
```

Comment:
AFAIK there's no case where streaming is set to false, or am I missing something? If so, we should wipe out the non streaming case altogether from the demo..

Reply:
Streaming is false for the first three cards, which get cached. It also could be false for browsers that don't support it... the AMP streaming code takes care of that for us, but I didn't know whether we should depend on that.

Reply:
ah ok, got it.